FirebaseDynamicLinks/Sources/FIRDynamicLinks.m

```diff
@@ -54,7 +54,7 @@
     @"com.google.appinvite.readDeeplinkAfterInstall";
 
 // We should only open url once. We use the following key to store the state in the user defaults.
-static NSString *const kFIRDLOpenURLKey = @"com.google.appinvite.openURL";
```

Comment:
Why remove the `static`?  There's no reason to pollute the app's namespace unless necessary.

Reply:
reverted